pkgs/by-name/ar/arrow-cpp/package.nix

```diff
@@ -80,14 +80,16 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "arrow-cpp";
-  version = "16.0.0";
+  version = "17.0.0";
 
-  src = fetchurl {
-    url = "mirror://apache/arrow/arrow-${finalAttrs.version}/apache-arrow-${finalAttrs.version}.tar.gz";
-    hash = "sha256-n0BRrpRzyXmR2a+AHi+UrjRVBncZyn+QuBN/nppwC40=";
+  src = fetchFromGitHub {
```

Comment:
Tarballs are better because they get backed up to tarballs.nixos.org

Reply:
I think `fetchFromGitHub` also downloads tarballs. I made the change because it is easier to test release candidates this way. I don't have a strong opinion tho.

Reply:
To my best understanding, it uses `fetchzip`, which, unlike `fetchurl` doesn't know the hash of the tarball itself, only hash of the unpacked contents. We don't expect apache repos to be removed, so it's probably okay this way too.